### PR TITLE
chore(GitHub): template for reporting issues with articles

### PR DESCRIPTION
**Pull request by majestic-owl448**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc. 

New template for reporting of articles needing fixing, discussion was started in May 8th mods meeting: https://www.freecodecamp.org/forum/t/notes-for-the-may-8th-guide-meeting/385787

This still needs a bit of discussion. I was told to do this in #39021 by @raisedadead .
I added some tags that I thought may work, but this could also be used for guide articles in the forum - or for that we want different way for user feedback?
I also don't really feel like the "Bug report" thing is correct. It may be.

Let's say this is to start talking about this.
And so there is also a way to report all the articles in \news that are still missing pieces.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/article-needs-fixing [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+name: 'Bug Report: Issues with \news or guide articles or documentation'
+about: Report issue with a specific article, like broken links, typos, missing parts, etc
+title: ''
+labels: 'status: needs help for triage, platform: news, platform: forum'
```
Comment (raisedadead) [inline .github/ISSUE_TEMPLATE/article-needs-fixing:5]:
```suggestion
labels: 'status: needs help for triage, type: bug'
```

The issue would need triage on which team (platform) should be addressing it. Also bug is the apt label.

File: .github/ISSUE_TEMPLATE/article-needs-fixing [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+NOTE: If you want to become an author of freecodecamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
+-->
+
+**Describe your problem and how to reproduce it:**
+
+
+**Add a Link to the page with the problem:**
+
+
+**How would you update it?**
```
Comment (raisedadead) [inline .github/ISSUE_TEMPLATE/article-needs-fixing:20]:
```suggestion
**Recommended fix, sugesstions (how would you update it?):**
```

File: .github/ISSUE_TEMPLATE/article-needs-fixing [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
```
Comment (raisedadead) [inline .github/ISSUE_TEMPLATE/article-needs-fixing:23]:
```suggestion
**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**

```
Newline

**Review by raisedadead — CHANGES_REQUESTED** [review]
Great starting point. Here is what you want to do next:

1. Apply the recommend suggestions.
2. Rename the file:

 	.github/ISSUE_TEMPLATE/general-report--issue-with-articles.md

If you need help you know where to reach us.

File: .github/ISSUE_TEMPLATE/article-needs-fixing (hunk 3 above)
Comment (Ieahleen) [reply .github/ISSUE_TEMPLATE/article-needs-fixing]: I have included the suggested changes, sorry for the double commit, github is confusing

**Comment by Ieahleen** [comment]
I have included the suggested changes, sorry for the double commit, github is confusing

File: .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+name: 'Bug Report: Issues with \news or guide articles or documentation'
+about: Report issue with a specific article, like broken links, typos, missing parts, etc
+title: ''
+labels: 'status: needs help for triage, type: bug'
+assignees: ''
+
+---
+
+<!-- 
+NOTE: If you want to become an author of freecodecamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
```
Comment (ojeytonwilliams) [inline .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md:11]:
```suggestion
NOTE: If you want to become an author on freeCodeCamp, you can find everything here: https://www.freecodecamp.org/news/developer-news-style-guide/
```
Nitpicks. 'on' makes is clear that they're producing content on the platform, not producing the platform itself.

File: .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+-->
+
+**Describe your problem and how to reproduce it:**
+
+
+**Add a Link to the page with the problem:**
+
+
+**Recommended fix, sugesstions (how would you update it?):**
```
Comment (ojeytonwilliams) [inline .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md:20]:
```suggestion
**Recommended fix, suggestions (how would you update it?):**
```

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Comment by ojeytonwilliams** [comment]
@Ieahleen don't worry about the double commit.  Typically a PR like this will get squashed to one commit, anyway.

**Review by ojeytonwilliams — DISMISSED** [review]
LGTM :+1:

File: .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
+---
+name: 'Bug Report: Issues with \news or guide articles or documentation'
```
Comment (raisedadead) [inline .github/ISSUE_TEMPLATE/general-report--issue-with-articles.md:2]:
```suggestion
name: 'Bug Report: Issues with /news or guide articles or documentation'
```

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Pull request merged**